docs/en/observability/ingest-traces.asciidoc

```diff
@@ -141,7 +144,14 @@ TIP: You can edit your APM integration settings if you need to change the APM Se
 or secret token to match your APM agents.
 
 --
+// this if directive includes the same file, but changes the file path based on where this guide is built
+ifndef::apm-integration-docs[]
 include::{apm-repo-dir}/legacy/tab-widgets/install-agents-widget.asciidoc[]
+endif::[]
+
+ifdef::apm-integration-docs[]
+include::{docdir}/legacy/tab-widgets/install-agents-widget.asciidoc[]
+endif::[]
```

Comment:
This is a bit of a mess, but it's the only way I could include this file in both the Observability Guide and APM Guide. Without the if directive, the file would error in one of the books.